docs/index.md

```diff
@@ -19,23 +19,48 @@ The software is maintained by the SuperDARN Data Analysis Working Group (DAWG) o
   - SuperDARN Data:
     * [Accessing Data](user_guide/data.md)
     * [Citing Data](user_guide/citing.md)
+    * [Filename Formats](references/general/filename.md)
+    * [Radar Identifier](references/general/radar_id.md)
   -  RST Tutorials 
-	  - Data Processing
-       * [RAWACF to FITACF](user_guide/make_fit.md)
-	     * [FITACF to GRID](user_guide/make_grid.md)
-	     * [GRID to MAP](user_guide/map_grid.md)
-	  - Plotting Data
+    - Data Processing
+         * [RAWACF to FITACF](user_guide/make_fit.md)
+         * [FITACF to GRID](user_guide/make_grid.md)
+         * [GRID to MAP](user_guide/map_grid.md)
+    - Plotting Data
          * [Range-time Plots](user_guide/time_plot.md)
          * [Field of View Plots](user_guide/fov_plot.md)
          * [Field Plots](user_guide/field_plot.md)
          * [Grid Plots](user_guide/grid_plot.md)
          * [Convection Plots](user_guide/map_plot.md)
          * [Customizing Colors](user_guide/colors.md)
   - Developers Guide
-    * [Copyrights and Licensing](developers_guide/copyright_license.md) 
+    - [Copyrights and Licensing](developers_guide/copyright_license.md) 
+    - Data Structures in C
```

Comment:
Need to add this to the `mkdoc.yml` file to have it appear on readthedocs side bar

Reply:
I was just noticing that the sidebar wasn't looking quite right, let me take a look at this and try to fix it.  Thanks!

Reply:
OK, I think I've got that fixed with b914de9.  Is it appropriate to add my info to the top of the `mkdoc.yml` file for authorship, copyright?

Reply:
Not sure but sure that or add under modifications

Reply:
OK, feel free to suggest something different if it doesn't look good.  There's no modifications here, but maybe you meant that I needed to add that to the comment block?  Sorry for asking some n00b questions, it's either been a while or first time trying this out.